CONTRIBUTING.md

```diff
@@ -3,22 +3,22 @@
 Welcome to the dMRIPrep repository!
 We're excited you're here and want to contribute.
 
-**Imposter syndrome disclaimer**: We want your help. No, really.
+**Imposter's syndrome disclaimer**: We want your help. No, really.
```

Comment:
```suggestion
**Impostor syndrome disclaimer**: We want your help. No, really.
```

Reply:
Also, add the proper attribution for the Impostor syndrome disclaimer. 

Taken from the original dmriprep:

> 
> The imposter syndrome disclaimer was originally written by
> [Adrienne Lowe](https://github.com/adriennefriend) for a [PyCon
> talk](https://www.youtube.com/watch?v=6Uj746j9Heo), and was
> adapted based on its use in the README file for the [MetPy
> project](https://github.com/Unidata/MetPy).

Reply:
Added in a footnote.